sheet_to_triples/field.py

```diff
@@ -164,14 +164,19 @@ def __contains__(self, item):
     def __eq__(self, other):
         return self.fields == other.fields
 
+    def col_exists(self, col):
+        # check if a column exists with a value that isn't None
+        return col in self.fields.keys() and self[col].exists
```

Comment:
This seems like a long spelling of:
```python
return self.fields.get(col) is not None
```
In fairness, the `exists()` implementation is wrong, it's still truthy like before, rather than being about not-`None`-ness.